packages/cloud_firestore_odm/cloud_firestore_odm_generator/lib/src/collection_generator.dart

```diff
@@ -413,6 +414,7 @@ extension on DartType {
         generic.isDartCoreString ||
         generic.isDartCoreBool ||
         generic.isDartCoreObject ||
+        generic.element?.kind == ElementKind.ENUM ||
```

Comment:
I don't see a test for `List<MyEnum>`. Could you add one? :smile:

Reply:
Remi,

First, I'm sorry for the delay, but I was trying to test the latest build against our project and it took me a while to figure out that we now seem to need a build.yaml in our own project in order to set the create_field_map=true. Otherwise it produces a bunch of errors. I didn't (currently) see this in the docs. I think if this is a requirement going forward it really needs to by highlighted. 

I also changed "dart pub get" to "flutter pub get" in melos.yaml/odm:generator_test_build_runner:build (It was giving me a lot of headaches)

I added List<SimpleEnum> to the Nested collection. Is that sufficient for the List<Enum> test?

Thanks for your support!

Reply:
Thanks for your effort!

About create_field_map, it should be optional. Maybe there's something I missed. I'll look into it

About your test, could you maybe move it to example/test_driver & add a test that performs a query?
Such as `collection.whereMyEnum(isEqualTo: MyEnum.value)` & maybe one for List too (although I think we're lacking `arrayContains` query for that.
There should be a few tests there that already do that which you could copy-paste

The goal would be to test that the generated work actually works, instead of simply compiling.